companion/lib/Surface/Controller.ts

```diff
@@ -1278,6 +1282,16 @@ export class SurfaceController extends EventEmitter<SurfaceControllerEvents> {
 					surface.setGroupId(newGroupId)
 					this.#attachSurfaceToGroup(surface)
 				}
+
+				// now copy the surfaceGroup into #surfaceGroups (does it matter that we don't copy `name`?)
+				// it appears that `surfaceHandlers` is empty if a surface is in a user-specified group
+				//  (note: I tried moving `&& group.surfaceHandlers.length > 0` to `group.#isAutoGroup` in Group.ts,
+				//  but it resulted in bogus groups being created when a device was attached -- and these groups only show up on next restart or on export.)
+				if (group && group.surfaceHandlers.length > 0) {
+					group.setGroupConfigValue('use_last_page', surfaceConfig.groupConfig.use_last_page)
```

Comment:
maybe this should be using the same logic as the `// Sync config` block near the start of this method?

Reply:
Not sure: `setGroupConfigValue` has a number of side-effects that I suspect we want to have. Especially when setting `last_page_id`

Reply:
OK. I must have been looking at the code cross-eyed when I wrote the previous reply.
Code updated as per your request.